instructors.html

```diff
@@ -27,21 +27,45 @@
         <div class="span10 offset1">
           <h1 class="title">Version Control with Git</h1>
           <h2 class="subtitle">Instructor's Guide</h2>
-<h2 id="legend">Legend</h2>
-<p>Wolfman and Dracula will work together in one project.</p>
-<h2 id="overall">Overall</h2>
+<p>Version control might be the most important topic we teach, but Git is definitely the most complicated tool. However, GitHub presently dominates the open software forge landscape, so we have to help novices learn just enough Git to feel they can and should learn more on their own.</p>
```

Comment:
I think it is important to let people know that there is a distinction between Git and GitHub. Although GitHub is a popular service, the availability of more private repos on other services warrants making a distinction between Git the tool and hosting services. I feel pretty strongly that if one mentions GitHub that other hosting service should be mentioned.

Possible text:
Git is a flexible and fast version control tool. Users have the option to working with the tool on just their computer or to use the tool in a distributed network of other remote computers and servers.

Many users use a cloud hosted version control service, like GitHub, for ease of collaboration and sharing of source code. While GitHub is currently the most popular version control service, Bitbucket, GitLab, and other services also provide hosting of repositories.

Many users like the convenience and collaboration that